spec/letsencrypt_spec.rb

```diff
@@ -37,4 +37,17 @@
       LetsEncrypt.register('example@example.com')
     end
   end
+
+  describe 'certificate_model' do
+    class OtherModel < LetsEncrypt::Certificate
+    end
+    before do
+      LetsEncrypt.config.certificate_model = 'OtherModel'
+      LetsEncrypt.stub(:certificate_model) { 'OtherModel'.constantize }
```

Comment:
```ruby
LetsEncrypt.stub(:certificate_model) { LetsEncrypt.config.certificate_model.constantize }
 ```

The stub method returns `'OtherModel'.constantize` but your assert is `'OtherModel'.constantize` too, they're no different and didn't check anything.